addons/website_sale/views/templates.xml

```diff
@@ -449,6 +449,18 @@
         </xpath>
     </template>
 
+    <template id="products_price" inherit_id="website_sale.products" active="False" customize_show="True" name="Filter by Prices" priority="12">
+        <xpath expr="//div[@id='products_grid_before']" position="before">
+            <t t-set="enable_left_column" t-value="True"/>
+        </xpath>
+        <xpath expr="//div[@id='products_grid_before']" position="inside">
+            <div id="o_wsale_price_range_option">
+                <strong>Catalog price</strong>
```

Comment:
Not a label ?

Reply:
I used the same layout as the attribute filter which uses strong element for the name of the attribute